client/src/featureform/client.py

```diff
@@ -0,0 +1,75 @@
+from typing import Union
+from .register import (
+    ResourceClient,
+    SourceRegistrar,
+    LocalSource,
+    SubscriptableTransformation,
+)
+from .serving import ServingClient
+
+
+class Client(ResourceClient, ServingClient):
```

Comment:
This is a hack which makes sense for now but when we deprecate we should fit all the code into here